apps/etterlatte-behandling/src/main/kotlin/behandling/Behandling.kt

```diff
@@ -42,6 +41,7 @@ sealed interface Behandling {
                 is Foerstegangsbehandling -> hentVirkningstidspunkt()
                 is ManueltOpphoer -> null
                 is Revurdering -> null
+                else -> null
```

Comment:
Er dette caset nødvendig?

Reply:
IntelliJ klager hvis ikke: 'when' expression must be exhaustive, add necessary 'else' branch

Reply:
Kunne du ikke bare forenklet til? 

`if (this is Foerstegangsbehandling) hentVirkningstidspunkt() else null`

Reply:
> IntelliJ klager hvis ikke: 'when' expression must be exhaustive, add necessary 'else' branch

Din IntelliJ er broken :D